libraries/AP_GPS/AP_GPS.h

```diff
@@ -241,6 +241,7 @@ class AP_GPS
     GPS_Status status(void) const {
         return status(primary_instance);
     }
+    GPS_Status min_status(void) const;
```

Comment:
We should probably add a comment above this method to clarify what its purpose in life is.